src/modules/extendedCallWindow/assets/vehicleListPermanentSearch.ts

```diff
@@ -0,0 +1,22 @@
+export default (): void => {
+    const vehicleTabs = document.querySelector('#tabs');
+    if (vehicleTabs) vehicleTabs.addEventListener('click', vehicleListChange);
+    //Initial expanding
+    vehicleListChange()
```

Comment:
Optional:
I would prefer putting these below the declaration of `vehicleListChange` but this is not necessary.